docs/user-guides/finetuner.md

```diff
@@ -7,6 +7,8 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
+We use `finetuner`==0.6.2, `clip-as-service`==0.8.0 and `docarray`==0.17.0 in this tutorial.
```

Comment:
```suggestion
We use `finetuner`==v0.6.2, `clip-as-service`==v0.8.0 and `docarray`==v0.17.0 in this tutorial.
```

Reply:
This tutorial only works on `clip-as-service==0.8.0` and `docarray=0.17.0`? I don't think so. This is misleading.

Reply:
We can loose the restriction on docarray. This feature requires cas>=0.6.0. And it should be the `clip_server` package